.travis.yml

```diff
@@ -1,8 +1,11 @@
 ---
+sudo: required
+services:
+    - docker
 language: python
 python: 2.7
 cache: pip
-install:
-  - "pip install vim-vint==0.3.9"
-script:
-  - "vint -s ."
+install: |
+    pip install vim-vint==0.3.9
+script: |
+    vint -s . && make test
```

Comment:
Maybe you can just run them line after line on Travis, and it will capture the error codes of all programs run.